src/main/index.ts

```diff
@@ -61,6 +61,10 @@ import { ExtensionsStore } from "../extensions/extensions-store";
 import { FilesystemProvisionerStore } from "./extension-filesystem";
 import { SentryInit } from "../common/sentry";
 
+// This has to be called before start using winton-based logger
+// For example, before any logger.log 
+SentryInit();
```

Comment:
Would you be able to expound a bit more as to why this has to be called before using winston?

Reply:
There will be an uncaught exception as in the PR description... I don't know why.

Reply:
Okay sounds good